public_html/wp-content/mu-plugins/wcorg-misc.php

```diff
@@ -442,18 +442,19 @@ function wcorg_central_modify_export_caps( $primitive_caps, $meta_cap ) {
 }
 add_filter( 'map_meta_cap', 'wcorg_central_modify_export_caps', 10, 2 );
 
+define( 'ERROR_RATE_LIMITING_DIR', '/tmp/error_limiting' );
+
 /**
  * Check and create filesystem dirs to manage rate limiting in error handling.
  * For legacy bugs we are doing rate limiting via filesystem. We would be investigating to see if we can instead use memcache to rate limit sometime in the future.
  *
  * @return bool Return true if file permissions etc are present
  */
 function init_error_handling() {
-	$error_dir = '/tmp/error_limiting';
-	if ( ! file_exists( $error_dir ) ) {
-		mkdir( $error_dir );
+	if ( ! file_exists( ERROR_RATE_LIMITING_DIR ) ) {
+		mkdir( ERROR_RATE_LIMITING_DIR );
```

Comment:
It's probably pretty safe to assume that `/tmp` exists, and that PHP wouldn't have the permissions to create it if it didn't, but in general it's a good habit to use `wp_mkdir_p`, since it's a bit safer. Not a big deal in this context, just FYI.